challenges/day7-issues/medium-task-3-3d-card/styles.css

```
@@ -0,0 +1,165 @@
+    .icon {
+        font-size: 3em;
+        margin-bottom: 15px;
+    }
+
+    .card-front h2 {
+        font-size: 1.5em;
+    }
+}
+
+/* Additional animation on card load */
+@keyframes cardEnter {
```

Stylelint (17.7.0)

[error] Expected keyframe name "cardEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)